src/bots/bot.ts

```diff
@@ -134,10 +135,12 @@ export default abstract class BotClient {
    */
   public async addSubscriber(channel: Channel, game: Game): Promise<boolean> {
     // Check if the bot can write to this channel
-    const permissions = await this.getUserPermissions(await this.getUser(), channel);
+    const user = await this.getUser();
+    const permissions = await this.getUserPermissions(user, channel);
 
     if (!permissions) {
       this.logger.error('Failed to get user permissions');
```

Comment:
Can the logger be passed to rollbar here as well?

Reply:
yeah but I don't know if I want to 🤔